fix: Django rq admin is broken: module 'django.utils.timezone' has no utc

makseq opened the pull request:
Fixes rq admin by bumping django-rq package
- https://github.com/HumanSignal/label-studio-sdk/pull/381 linked by [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/12627881994)

makseq committed:
fix: Django rq admin is broken: module 'django.utils.timezone' has no attribute 'utc'

jombooth commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/12604705895)
> Successfully merged:  3 files changed, 39 insertions(+), 19 deletions(-)

jombooth commented:
https://github.com/rq/django-rq/blob/master/CHANGELOG.md <- I see what you did here, staying on RQ 1 and django-rq 2 makes sense to me. LGTM once conflicts fixed

jombooth committed:
Merge branch 'develop' into 'fb-optic-1488'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12604705895

robot-ci-heartex commented:
Follow Merge downstream workflow has been failed.
> [Workflow run](https://github.com/HumanSignal/label-studio-enterprise/actions/runs/12604728601)

jombooth commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/12625693006)
> Successfully merged: Already up to date.

jombooth committed:
raise lowest usable version of rq

jombooth committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12627881994

jombooth committed:
pin to pre-2 poetry-core

jombooth committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12636494083
